styles.css

```diff
@@ -26,4 +26,12 @@ li a {
   padding: 20px 16px;
   text-decoration: none;
   font-family: "Courier New", Courier, monospace
+}
+
+section {
+  max-height: 800px;
+  max-width: 800px;
```

Comment:
ok, i need hard parameters here... not that cool of a programmer yet to have discretion haha!